main.py

```
@@ -1,44 +1,88 @@
+@app.route("/postback")
+def postback():
+    user_id = request.args.get("userid")
+    payout = request.args.get("payout")
+    
+    if user_id and payout:
+        cursor.execute("UPDATE users SET balance = balance + ? WHERE user_id=?", (float(payout), int(user_id)))
+        conn.commit()
+        
+        try:
+            bot_app.bot.send_message(chat_id=int(user_id), text=f"🎉 New Earning: ${payout}")
+        except:
```

Ruff (0.15.1)

[error] Do not use bare `except`

(E722)